### PR TITLE
quick fix file loading done

### PR DESCRIPTION
**Pull request by lonelu**
quick fix file loading. For glyco RAM issue.

### CODE REVIEW

**Comment by lonelu** [comment]
quick fix file loading. For glyco RAM issue.

**Review by acesnik — APPROVED** [review]

**Review by trishorts — APPROVED** [review]

**Review by rmmiller22 — APPROVED** [review]

**Comment by rmmiller22** [comment]
Update the branch and I will merge in PR

**Comment by rmmiller22** [comment]
This pull request is being reverted. Jenkins produced 0 PSMs once it (and another PR) were merged. You can re-open/ re-create this PR- and can be merged again once issues are identified and resolved.

**Pull request merged**
